### DISCUSSION TITLE
TShock - error with nickname

### DISCUSSION THREAD

**Discussion by Neu3BecTHo** [Pryaxis/TShock#2628] [Q&A] [answered]
Hello. Now I have a problem with nicknames. Previously, everything was fine, i.e. nicknames containing Russian characters could normally log on to the server and play. But now, unfortunately, such players simply cannot enter. They register, then try to log in... mistake.
![image](https://user-images.githubusercontent.com/87427271/167294982-b2866741-4089-4813-8762-84fcf15a057a.png)
it also knocks out this (I think it's because all Russian nicknames turn into ?????)
![image](https://user-images.githubusercontent.com/87427271/167294997-629a4ad7-3ff9-4c6c-aee5-3fc01b3199ea.png)

**Comment by Arthri** [comment, marked as answer]
Hm, I'm not entirely sure but I suspect it has something to do with the character set in the MySQL database, what character set are you using?

**Reply by Neu3BecTHo** [reply]
![image](https://user-images.githubusercontent.com/87427271/167295884-b0c8c610-a313-4a92-9989-d20aca586862.png)
this?

**Reply by Neu3BecTHo** [reply]
Yes, it's about the encoding on the server.
